dialect/sql/schema/migrate.go

```diff
@@ -10,137 +10,102 @@ import (
 	"errors"
 	"fmt"
 	"math"
+	"net/url"
```

Comment:
It's really hard to do a good code review and ignore irrelevant changes when there are files renaming/deleting in the same PR. The change is too big to distinguish what is new and what is not.